lib/TimezoneOffsetUtil.js

```diff
@@ -69,6 +69,16 @@ module.exports = function(timezone, mostRecent, changes) {
     else {
       conversionOffset += difference.rawDifference;
     }
+
+    // if timezone offset is invalid, move to conversion offset until valid
+    while (timezoneOffset > 840) {
```

Comment:
Might want to consider using well-named constants rather than the raw numbers, which just look like secret values unless you understand that min/max time zone offset is -12/14 hours and `timezoneOffset` is in minutes. This would apply to the `840`, `-720`, and `1440`. Up to you, though.